server/lib/parties/party-service.ts

```diff
@@ -191,12 +186,10 @@ export default class PartyService {
       )
     }
 
-    if (!party.invites.has(target.id)) {
-      throw new PartyServiceError(
-        PartyServiceErrorCode.InvalidAction,
-        "Can't remove invite for a user that wasn't invited",
-      )
-    }
+    // If an invite doesn't exist for a user it most likely means that the user has already declined
```

Comment:
This in particular feels like something that would be fine as an error case with the semantics change I mentioned previously. You can decide whether you'd like to do that semantics change in this PR or separately.

Reply:
The reason why I removed this error case is because the user was removed from the invites list when they decline, so it could potentially leak information about whether they declined or not. Which made me now think that we should probably not remove the user from the invites list if they decline and this error case can then stay :). See the comment below (above?) where I mentioned this as well.

Reply:
Right, I think we're trying to say the same thing here (this error is fine to send provided the invite list becomes an "allowed to join" list), sorry if that wasn't clear :)